remote_attestation/rust/src/handshaker.rs

```diff
@@ -51,7 +51,7 @@ impl Default for ClientHandshakerState {
     }
 }
 
-impl std::fmt::Display for ClientHandshakerState {
+impl std::fmt::Debug for ClientHandshakerState {
```

Comment:
Now that this is changed to `Debug`, why not use derive?

Reply:
Because `assert_matches` requires `Debug` to be implemented.

Reply:
Is not it possible to use `#[derive(Debug)]` instead of implementing `Debug` manually?

Reply:
No, because some of the states contain `KeyNegotiator` and `Encryptor`, and printing them is prohibited.

Reply:
Can you please add a comment explaining this?